test/test_nn.py

```diff
@@ -2331,7 +2331,7 @@ def test_prob_module(self, interaction, return_log_prob, map_names):
                     },
                 }
             },
-            [3],
+            batch_size=3,
```

Comment:
```suggestion
            batch_size=(3,),
```

Reply:
would type checking be considered for `tensordict` ?